test/integration/che/che-steps.go

```diff
@@ -42,13 +42,13 @@ func FeatureContext(s *godog.Suite) {
 	s.Step(`^che api endpoint should not be empty$`, cheAPIRunner.cheApiEndpointShouldNotBeEmpty)
 	s.Step(`^user tries to get the stacks information$`, cheAPIRunner.weTryToGetTheStacksInformation)
 	s.Step(`^the stacks should not be empty$`, cheAPIRunner.theStacksShouldNotBeEmpty)
-	s.Step(`^starting a workspace with stack "([^"]*)" succeeds$`, cheAPIRunner.startingWorkspaceWithStackSucceeds)
+	s.Step(`^starting a workspace with stack "([^"]*)" succeeds within "(\d+(?:ms|s|m))"$`, cheAPIRunner.startingWorkspaceWithStackSucceeds)
 	s.Step(`^workspace should have state "([^"]*)"$`, cheAPIRunner.workspaceShouldHaveState)
 	s.Step(`^importing the sample project "([^"]*)" succeeds$`, cheAPIRunner.importingTheSampleProjectSucceeds)
 	s.Step(`^workspace should have (\d+) project$`, cheAPIRunner.workspaceShouldHaveProject)
 	s.Step(`^user runs command on sample "([^"]*)"$`, cheAPIRunner.userRunsCommandOnSample)
 	s.Step(`^exit code should be (\d+)$`, cheAPIRunner.exitCodeShouldBe)
-	s.Step(`^stopping a workspace succeeds$`, cheAPIRunner.stoppingWorkspaceSucceeds)
+	s.Step(`^stopping a workspace succeeds within "(\d+(?:ms|s|m))"$`, cheAPIRunner.stoppingWorkspaceSucceeds)
 	s.Step(`^workspace is removed$`, cheAPIRunner.workspaceIsRemoved)
 	s.Step(`^workspace removal should be successful$`, cheAPIRunner.workspaceRemovalShouldBeSuccessful)
 }
```

Comment:
@agajdosi This comment is not for this PR but for future. I think the Che specific steps should be also be moved to `testsuite/testsuite.go`.  It is also way good to have all the steps in one place from long-term maintenance/readablity point of view.